content/en/blog/grpc-on-arm64.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Running gRPC and Protobuf on ARM64 (on Linux)
+date: 2021-06-20
+spelling: Tattermusch
+author:
+  name: Jan Tattermusch
+  link: https://github.com/jtattermusch
+---
+
+ARM processors have recently been gaining importance in many areas of compute, including those that were traditionally considered to be an x86_64-only domain. Thanks to the momentum around the ARM ecosystem, we can expect adoption of ARM platforms to grow significantly. The same is true for providing software that supports ARM-based platforms.
+
+Since the main purpose of gRPC and protocol buffers is to interconnect distributed systems, their role in supporting ARM compute is especially important. With the emergence of ARM Cloud computing, we expect many systems will actually be hybrid compositions of x86 and ARM servers, mixing and matching the qualities of each ecosystem as needed. gRPC and protocol buffers are the ideal building blocks that can enable users to build systems that span multiple architectures seamlessly.
+
+To meet the very high demand from gRPC users to support ARM, the gRPC team has decided to support selected ARM-based platforms officially and fully. Some time ago, we started an effort to test everything and fix any problems we encounter. **Today, we are happy to announce that gRPC and protocol buffers implementations in C++, C#, Go, Java, Node, PHP, Python and Ruby are ready for production workloads for ARM64 Linux** (see more details below).
+
+The current state of things is best described by a list of general areas we completed:
+
+- **Bugfixes/improvements:** We put a number of fixes in place to make sure gRPC and protobuf work reliably on ARM64 Linux.
+- **Package & Distribution:** For the languages that provide binary architecture-specific packages, we added ARM64 Linux packages, and we started publishing them on every release as part of our standard release process. Having ready-to-use binary packages greatly improves the developer experience. For languages that don't ship using binary packages, we tested that the build works as expected and that one can install gRPC and protobuf on ARM64 Linux without problems.
+- **Continuous Testing:** We invested significant resources setting up continuous testing ensuring that gRPC and protobuf are fully tested and prevent any future regressions. Testing large projects with many tests and support for multiple languages was definitely a challenge, especially since the open source ecosystem for testing on ARM architecture is still in its infancy, but we successfully tested gRPC and protobuf with a combination of cross-compilation, emulated tests and running tests on real hardware.
+
+## Overview of gRPC and Protobuf support on ARM64 Linux
+
+In the following table you can see the detailed status broken down by language. Each entry summarizes the support level for both gRPC and Protobuf on ARM64 Linux.
+
+| Language        | continuously tested | distribution/packages | additional info |
```

Comment:
@chalin, in the rendered version of the blogpost the wrapping for "language" and "continuously tests" is weird? What is the best way to prevent that?
Also, is the a best practice for adding a color background for table cells? (I'd like to make background of some cells green to make the level of support more obvious.